readthedocs/templates/core/project_list_detailed.html

```diff
@@ -29,24 +29,15 @@
     <span class="right-menu quiet">{% trans "No builds" %}</span>
     {% endif %}
 
-    {% if project.has_good_build %}
-    <span class="dropdown" style="position:absolute; right:0px; top:0px;">
-      <span>
-          <a href="{{ project.get_docs_url }}">{% trans "View Docs" %}</a>
-        <a href="#">&#x25BC;</a>
-      </span>
-      <ul>
-          <li><input type="search" placeholder="{% trans "Search" %}&hellip;" /></li>
-          {% for version in project.ordered_active_versions reversed %}
-              <li><a href="{{ version.get_absolute_url }}">{{ version.slug }}</a></li>
-          {% endfor %}
-      </ul>
-    </span>
-    {% else %}
     <ul class="module-item-menu">
-      <li><a href="{{ project.get_builds_url }}">{% trans "No Docs" %}</a></li>
+      <li>
+        {% if project.has_good_build %}
+          <a href="{{ project.get_docs_url }}">{% trans "View Docs" %}
+          {% else %}
+            <a href="{{ project.get_builds_url }}">{% trans "No Docs" %}</a>
+          {% endif %}
```

Comment:
nit: `else` and `endif` indentation is wrong.

Reply:
Actually it was missing a `</a>` :/